iXBRLViewerPlugin/viewer/src/js/report.js

```diff
@@ -153,4 +153,6 @@ iXBRLReport.prototype.getConcept = function(name) {
 }
 
 
-
+iXBRLReport.prototype.getRoleLabel = function(rolePrefix, viewerOptions) {
+    return this.data.roleDefs[rolePrefix]["en"];
```

Comment:
Can this be enhanced to use user-selected language?

Reply:
At present the Arelle plugin only provides a single "en" label because it's using the role "definition" which can't be localised.  It is possible to attach localised generic labels to role types, but the plugin doesn't currently look for these, and I don't think that labels are commonly provided in this manner in real taxonomies.  I've added a comment in ef61de8 to explain the hard-coding.

Reply:
👍